package.json

```diff
@@ -35,8 +36,14 @@
     "xhr": "2.2.0"
   },
   "devDependencies": {
-    "babel": "^5.2.2",
-    "babelify": "^6.0.1",
+    "babel-cli": "^6.11.4",
+    "babel-plugin-inline-json-config-values": "^1.0.0",
```

Comment:
There are two outstanding PRs against this repo right now. I'm going to wait a week or so before publishing my own version to use instead:
https://github.com/mwilliams-change/babel-plugin-inline-json-config-values/pull/2
https://github.com/mwilliams-change/babel-plugin-inline-json-config-values/pull/3